src/AutoRest.CSharp/Common/Output/Expressions/KnownValueExpressions/XAttributeExpression.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Xml.Linq;
+using AutoRest.CSharp.Common.Output.Expressions.ValueExpressions;
+
+namespace AutoRest.CSharp.Common.Output.Expressions.KnownValueExpressions
+{
+    internal sealed record XAttributeExpression(ValueExpression Untyped) : TypedValueExpression<XAttribute>(Untyped)
```

Comment:
this is not a new class. This is an existing record previously defined in the same file as another record.
This change just extract this to make it to have its own file